Backend/Interview.Domain/Rooms/Service/RoomService.cs

```diff
@@ -164,6 +165,59 @@ public async Task<IPagedList<RoomPageDetail>> FindPageAsync(
         });
     }
 
+    public async Task<List<RoomCalendarItem>> GetCalendarAsync(RoomCalendarRequest filter, CancellationToken cancellationToken = default)
+    {
+        var currentUserId = _currentUserAccessor.UserId;
+
+        var queryable = _db.Rooms
+            .Include(room => room.Participants)
+            .Where(room => room.Participants.Any(participant => participant.UserId == currentUserId));
+
+        if (filter.StartDateTime is not null)
+        {
+            queryable = queryable.Where(e => filter.StartDateTime <= e.ScheduleStartTime);
```

Comment:
Shouldn't we take DateTimeOffset into account when searching for records and translate the passed time from frontend to utc first?